From fb806e5b1776908d9eff71a789de796502f635ce Mon Sep 17 00:00:00 2001
From: Magnus Damm <damm@opensource.se>
Date: Tue, 1 Oct 2013 17:13:26 +0900
Subject: ARM: shmobile: Use SMP on Koelsch

Enable r8a7791 SMP support code on the Koelsch board.

Signed-off-by: Magnus Damm <damm@opensource.se>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit b6d5a1b1c3476225e897a2f706c0e7eca7b05984)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/mach-shmobile/board-koelsch.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-shmobile/board-koelsch.c b/arch/arm/mach-shmobile/board-koelsch.c
index ea5846e2e88c..ace1711a6cd8 100644
--- a/arch/arm/mach-shmobile/board-koelsch.c
+++ b/arch/arm/mach-shmobile/board-koelsch.c
@@ -39,6 +39,7 @@ static const char * const koelsch_boards_compat_dt[] __initconst = {
 };
 
 DT_MACHINE_START(KOELSCH_DT, "koelsch")
+	.smp		= smp_ops(r8a7791_smp_ops),
 	.init_early	= r8a7791_init_early,
 	.init_machine	= koelsch_add_standard_devices,
 	.init_time	= rcar_gen2_timer_init,
-- 
1.8.5.rc3

